src/pages/guides/using/aem/aem_skyline_install.md

```diff
@@ -15,9 +15,9 @@ The Adobe I/O Events integration is currently being exposed through an integrati
 3. [prepare the Adobe I/O Events service user keystore](aem_keystore_setup.md) 
```

Comment:
Could you check the link in point 2 for accessing git repo?
https://docs.adobe.com/content/help/en/experience-manager-cloud-manager/using/managing-code/accessing-git.html
Looks like it does not exist anymore. Can we fix that?